### ISSUE TITLE
[Issue]: Missing hsaKmtPcSamplingSupport and hsaKmtAllocMemoryAlign symbols in 6.2.0 release

### ISSUE THREAD

**Issue by AngryLoki** [ROCm/ROCT-Thunk-Interface#106]
### Problem Description

Building `tests/kfdtest` in rocm-6.2.0 branch fails with:

```
FAILED: kfdtest 
: && /usr/lib/llvm/18/bin/clang++ -march=native -O2 -pipe -fdiagnostics-color=always -flto=thin -Werror=odr -Werror=strict-aliasing -O2 -Wl,-O1 -Wl,--as-needed -Wl,-z,pack-relative-relocs -Wl,-O2 -Wl,--as-needed -fdiagnostics-color=always CMakeFiles/kfdtest.dir/gtest-1.6.0/gtest-all.cpp.o CMakeFiles/kfdtest.dir/src/AqlQueue.cpp.o CMakeFiles/kfdtest.dir/src/BasePacket.cpp.o CMakeFiles/kfdtest.dir/src/BaseDebug.cpp.o CMakeFiles/kfdtest.dir/src/BaseQueue.cpp.o CMakeFiles/kfdtest.dir/src/Dispatch.cpp.o CMakeFiles/kfdtest.dir/src/GoogleTestExtension.cpp.o CMakeFiles/kfdtest.dir/src/IndirectBuffer.cpp.o CMakeFiles/kfdtest.dir/src/Assemble.cpp.o CMakeFiles/kfdtest.dir/src/ShaderStore.cpp.o CMakeFiles/kfdtest.dir/src/LinuxOSWrapper.cpp.o CMakeFiles/kfdtest.dir/src/PM4Packet.cpp.o CMakeFiles/kfdtest.dir/src/PM4Queue.cpp.o CMakeFiles/kfdtest.dir/src/RDMAUtil.cpp.o CMakeFiles/kfdtest.dir/src/SDMAPacket.cpp.o CMakeFiles/kfdtest.dir/src/SDMAQueue.cpp.o CMakeFiles/kfdtest.dir/src/KFDBaseComponentTest.cpp.o CMakeFiles/kfdtest.dir/src/KFDMultiProcessTest.cpp.o CMakeFiles/kfdtest.dir/src/KFDTestMain.cpp.o CMakeFiles/kfdtest.dir/src/KFDTestUtil.cpp.o CMakeFiles/kfdtest.dir/src/KFDTestUtilQueue.cpp.o CMakeFiles/kfdtest.dir/src/KFDOpenCloseKFDTest.cpp.o CMakeFiles/kfdtest.dir/src/KFDTopologyTest.cpp.o CMakeFiles/kfdtest.dir/src/KFDMemoryTest.cpp.o CMakeFiles/kfdtest.dir/src/KFDLocalMemoryTest.cpp.o CMakeFiles/kfdtest.dir/src/KFDEventTest.cpp.o CMakeFiles/kfdtest.dir/src/KFDQMTest.cpp.o CMakeFiles/kfdtest.dir/src/KFDCWSRTest.cpp.o CMakeFiles/kfdtest.dir/src/KFDExceptionTest.cpp.o CMakeFiles/kfdtest.dir/src/KFDGraphicsInterop.cpp.o CMakeFiles/kfdtest.dir/src/KFDPerfCounters.cpp.o CMakeFiles/kfdtest.dir/src/KFDDBGTest.cpp.o CMakeFiles/kfdtest.dir/src/KFDGWSTest.cpp.o CMakeFiles/kfdtest.dir/src/KFDIPCTest.cpp.o CMakeFiles/kfdtest.dir/src/KFDASMTest.cpp.o CMakeFiles/kfdtest.dir/src/KFDEvictTest.cpp.o CMakeFiles/kfdtest.dir/src/KFDHWSTest.cpp.o CMakeFiles/kfdtest.dir/src/KFDPerformanceTest.cpp.o CMakeFiles/kfdtest.dir/src/KFDPMTest.cpp.o CMakeFiles/kfdtest.dir/src/KFDSVMRangeTest.cpp.o CMakeFiles/kfdtest.dir/src/KFDSVMEvictTest.cpp.o CMakeFiles/kfdtest.dir/src/KFDRASTest.cpp.o CMakeFiles/kfdtest.dir/src/KFDPCSamplingTest.cpp.o CMakeFiles/kfdtest.dir/src/RDMATest.cpp.o -o kfdtest -L/var/tmp/portage/dev-libs/roct-thunk-interface-6.2.0/work/ROCT-Thunk-Interface-rocm-6.2.0_build -Wl,-rpath,/var/tmp/portage/dev-libs/roct-thunk-interface-6.2.0/work/ROCT-Thunk-Interface-rocm-6.2.0_build:/usr/lib/llvm/18/lib64  -lhsakmt  -L/usr/lib64  -ldrm  -L/usr/lib64  -ldrm_amdgpu  /usr/lib/llvm/18/lib64/libLLVM.so.18.1+libcxx  -lpthread  -lm  -lstdc++  -lrt  -lnuma && :
ld.lld: error: undefined symbol: hsaKmtPcSamplingSupport
>>> referenced by KFDPCSamplingTest.cpp
>>>               CMakeFiles/kfdtest.dir/src/kfdtest.lto.KFDPCSamplingTest.cpp.o:(KFDPCSamplingTest_BasicTest_Test::TestBody())
>>> referenced by KFDPCSamplingTest.cpp
>>>               CMakeFiles/kfdtest.dir/src/kfdtest.lto.KFDPCSamplingTest.cpp.o:(KFDPCSamplingTest_MultiThreadPcSamplingTest_Test::TestBody())
>>> referenced by KFDPCSamplingTest.cpp
>>>               CMakeFiles/kfdtest.dir/src/kfdtest.lto.KFDPCSamplingTest.cpp.o:(KFDPCSamplingTest_MultiProcPcSamplingTest_Test::TestBody())
>>> referenced 1 more times
```

Also build or rocminfo-6.2.0 fails in https://github.com/ROCm/ROCR-Runtime/blob/rocm-6.2.0/src/core/runtime/runtime.cpp#L2978 with:
```
: && /usr/bin/x86_64-pc-linux-gnu-g++ -O2 -pipe -Wl,-O1 -Wl,--as-needed -Wl,-z,pack-relative-relocs CMakeFiles/rocminfo.dir/rocminfo.cc.o -o rocminfo  /usr/lib64/libhsa-runtime64.so.1.14.0 && :
/usr/libexec/gcc/x86_64-pc-linux-gnu/ld: /usr/lib64/libhsa-runtime64.so.1.14.0: undefined reference to `hsaKmtAllocMemoryAlign'
```

Indeed in https://github.com/ROCm/ROCT-Thunk-Interface/commit/e82b8dfe80482aa11efe0ee9eecdf7f0f0a769a6#diff-d22eb13da9b67be9adefa9d67385cb732b07c58ad42ab1b2930f5ef2cd40a21d some new symbols were added, except for one - `hsaKmtPcSamplingSupport`. Please add it.

```diff
--- a/src/libhsakmt.ver
+++ b/src/libhsakmt.ver
@@ -82,6 +82,8 @@ hsaKmtWaitOnMultipleEvents_Ext;
 hsaKmtReplaceAsanHeaderPage;
 hsaKmtReturnAsanHeaderPage;
 hsaKmtGetAMDGPUDeviceHandle;
+hsaKmtPcSamplingSupport;
+hsaKmtAllocMemoryAlign;
 hsaKmtPcSamplingQueryCapabilities;
 hsaKmtPcSamplingCreate;
 hsaKmtPcSamplingDestroy;

```

### Operating System

Gentoo

**Comment by AngryLoki** [comment]
Note: updated description to mention hsaKmtAllocMemoryAlign too.

**Comment by kentrussell** [comment]
Aiming to get this into 6.2.1.

**Comment by zichguan-amd** [comment]
Fixed in 6.2.1
